src/LEGO.AsyncAPI/Models/AsyncApiSecurityScheme.cs

```diff
@@ -9,6 +9,92 @@ namespace LEGO.AsyncAPI.Models
 
     public class AsyncApiSecurityScheme : IAsyncApiSerializable, IAsyncApiExtensible
     {
+        public static AsyncApiSecurityScheme UserPassword(string description = null) => new()
+        {
+            Type = SecuritySchemeType.UserPassword,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme ApiKey(ParameterLocation @in, string description = null) => new()
+        {
+            Type = SecuritySchemeType.ApiKey,
+            Description = description,
+            In = @in,
+        };
+
+        public static AsyncApiSecurityScheme X509(string description = null) => new()
+        {
+            Type = SecuritySchemeType.X509,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme SymmetricEncryption(string description = null) => new()
+        {
+            Type = SecuritySchemeType.SymmetricEncryption,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme AsymmetricEncryption(string description = null) => new()
+        {
+            Type = SecuritySchemeType.AsymmetricEncryption,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme HttpApiKey(string name, ParameterLocation @in, string description = null) => new()
+        {
+            Type = SecuritySchemeType.HttpApiKey,
+            Description = description,
+            Name = name,
+            In = @in,
+        };
+
+        public static AsyncApiSecurityScheme Http(string scheme, string? bearerFormat = null, string description = null) => new()
+        {
+            Type = SecuritySchemeType.Http,
+            Description = description,
+            Scheme = scheme,
+            BearerFormat = bearerFormat,
+        };
+
+        public static AsyncApiSecurityScheme OAuth2(AsyncApiOAuthFlows flows, string[] scopes = null, string description = null) => new()
```

Comment:
I dont think null scopes can technically be provided here.
Either empty list or list of some. be sure to new it up, if scopes is null at least (or make a null check and throw)